notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala

```diff
@@ -124,7 +140,11 @@ class DistCopyMapper extends Mapper[NullWritable, Mapping, NullWritable, NullWri
           outputStream => from.withStreamMultipart(
             Bytes(partSize)
             , in => {
-              val counted = DownloadInputStream(in, i => totalBytesDownloaded.increment(i))
+              val counted = DownloadInputStream(in, i => {
+                totalBytesDownloaded.increment(i)
+                totalMegaBytesDownloaded.increment(i / 1024 / 1024)
+                totalGigaBytesDownloaded.increment(i / 1024 / 1024 / 1024)
```

Comment:
Sorry, I got caught into thinking that we were just losing precision. The fix is here #93